models/packages.go

```diff
@@ -174,6 +176,20 @@ func (p Package) FormatChangelog() string {
 	return strings.Join(buf, "\n")
 }
 
+func (p Package) In3rdPartyRepo(family string) bool {
```

Comment:
__[Golint]__

exported method Package.In3rdPartyRepo should have comment or be unexported